content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -424,3 +427,21 @@ namespace.
 
 When each team has their own mesh, cross-mesh communication follows the
 concepts described in the [multiple meshes](#multiple-meshes) model.
+
+## Mesh expansion
+
+In same cases you might have to deploy some of the services on virtual machines or bare metal computers. The
+reasons for deployments on VMs/bare metal computers could be special hardware requirements, tighter isolation, tighter integration with the host's OS, legacy deployments.
+
+Having some of the services on VMs/bare metal, you may want to connect the services into a mesh and benefit from all the
+service mesh features: traffic management, security, observability, and others. You can deploy the Istio control plane(s)
+on some of your clusters and expand the mesh of the cluster(s) by adding VMs/bare metal computers into the mesh.
+
+The VMs/bare metal computers can share the same network with the clusters of the mesh, or they can be on
```

Comment:
This paragraph seems redundant with other concepts. I think we can lose it.

Reply:
Done in https://github.com/istio/istio.io/pull/6209/commits/3f4ce88594fb82b137a62288eea4f7a30f72a0de.